### DISCUSSION TITLE
关于自动跑比赛的情况

### DISCUSSION THREAD

**Discussion by aztuo** [NateScarlet/auto-derby#147] [General]
我注意到脚本会选择OP级的比赛来进行，OP级的收益过低，是否应该避免掉啊

**Comment by NateScarlet** [comment]
OP不跑粉丝数会不够跑G1 或者固有技能不升级 按时间有通用粉丝数要求 越晚差得越多就会越倾向于跑比赛

https://github.com/NateScarlet/auto-derby/blob/6d01292949bfd12946a91ca7cc501c6813701460/auto_derby/single_mode/race.py#L612-L630

如果不跑高级比赛检查是不是你属性太低脚本认为你跑不了第一名 debug 有每个比赛的预测名次

如果要自定义赛程见 https://github.com/NateScarlet/auto-derby/wiki/FAQ#%E5%A6%82%E4%BD%95%E8%87%AA%E5%AE%9A%E4%B9%89%E8%B5%9B%E7%A8%8B

**Reply by sixZI** [reply]
你好，我发现了会有提示粉丝数不够但是不去跑比赛，最后一战一胜的情况，想看一下相关代码的部分，确认一下是我插件的问题还是原脚本的问题。能否指点一下相关代码的位置？

**Reply by NateScarlet** [reply]
https://github.com/NateScarlet/auto-derby/blob/7abde423fffb532fa6195e2202212cc603351486/auto_derby/single_mode/commands/training.py#L36-L43

**Reply by NateScarlet** [reply]
你是不是把休息分数定太高了让比赛不如休息

**Reply by sixZI** [reply]
感谢回复
我是夏休前休息分数调的比较高其余时候休息的分数没有改动。
一战一胜那一次是在养米浴，要参加皐月賞前的一场比赛，需求粉丝数1500人，然后出现了这个问题···

**Reply by sixZI** [reply]
有没有可能是当时马的属性比较弱，脚本认为他拿不到前3所以没去比赛？

**Reply by NateScarlet** [reply]
再弱也不会认为OP拿不了前三吧 你自己看debug日志的预测名次

**Reply by sixZI** [reply]
> 再弱也不会认为OP拿不了前三吧 你自己看debug日志的预测名次

搞明白了，是我调低了OP的分数，所以他没去比，我赛程定义的有问题。
